src/main/java/org/opencds/cqf/tooling/processor/CqlProcessor.java

```diff
@@ -186,6 +187,11 @@ public CqlSourceFileInformation getFileInformation(String filename) {
         }
 
         if (!fileMap.containsKey(filename)) {
+            for (Map.Entry<String, CqlSourceFileInformation> entry: fileMap.entrySet()) {
+                if (filename.equalsIgnoreCase(entry.getKey())) {
+                    System.out.println(String.format("WARNING: File with a similar name but different casing was found. File found: '%s'", entry.getKey()));
```

Comment:
I think this should use Log4J logging rather than **System.out.print()**...

Reply:
Agreed, switched this to use the logger that's part of the base processor